cmd/vm.go

```diff
@@ -87,4 +56,12 @@ func init() {
 	if err != nil {
 		_ = fmt.Errorf("kvm option not supplied")
 	}
+
+	// options for vm generic commands related to HTTP Request
+	vmCmd.PersistentFlags().StringVar(&HttpIP, "http-ip", "127.0.0.1", "IP of kvm-service")
+	vmCmd.PersistentFlags().StringVar(&HttpPort, "http-port", "8000", "Port of kvm-service")
+	vmCmd.PersistentFlags().BoolVar(&IsNonK8sEnv, "non-k8s", false, "Enable if non-k8s environment/control-plane")
```

Comment:
Since it is already under the `vm` subcommand. Non k8s flag doesn't really make sense imo. Perhaps using control plane or kvm... Not sure what would be better here but not nonk8s cause nonk8s applies to both systemd and control plane.

Reply:
Since we are using kvm-service, we can name that flag as kvms instead of non-k8s. That will be different. Please suggest

Reply:
Sure.